partcad/src/partcad/cache.py

```
@@ -0,0 +1,94 @@
+
+    def _needs_write_data(self, data_len: int) -> bool:
+        """Check if object needs to be written to cache."""
+        # Make an exception for 1 byte objects to cache test results
+        if data_len >= 2 and data_len < user_config.cache_min_entry_size:
+            # This object is too small to cache
+            return False
+        if data_len > user_config.cache_max_entry_size:
+            # This object is too big to cache
+            return False
+
+        return True
```

Ruff (0.8.2)

Return the negated condition directly

(SIM103)